sunpy/map/header_helper.py

```diff
@@ -124,8 +124,8 @@ def make_fitswcs_header(data, coordinate, reference_pixel: u.pix = None,
     meta_wcs['crval1'], meta_wcs['crval2'] = (coordinate.spherical.lat.to_value(meta_wcs['cunit1']),
                                               coordinate.spherical.lon.to_value(meta_wcs['cunit2']))
 
-    meta_wcs['crpix1'], meta_wcs['crpix2'] = (reference_pixel[0].to_value(u.pixel),
-                                              reference_pixel[1].to_value(u.pixel))
+    meta_wcs['crpix1'], meta_wcs['crpix2'] = (reference_pixel[0].to_value(u.pixel) + 1,
+                                              reference_pixel[1].to_value(u.pixel) + 1)
```

Comment:
what if someone assumes this already when they add in what the reference pixel should be?

Reply:
We need to document this clearer if we expect this to be 1 indexed or 0 indexed. We definitely need to fix it if the user doesn't specify it. I am not sure however if we should expect the user to give us these values 0 or 1 indexed.

This is the only kwarg we need to make this decision on though.